operations/vouchers.md

```diff
@@ -54,85 +57,87 @@ Returns all rate vouchers filtered by [Service](services.md#service), voucher co
-            "ValidityEndUtc": "2020-10-09T22:00:00Z",
-            "CreatedUtc": "2020-10-09T15:08:14Z",
-            "UpdatedUtc": "2020-10-09T15:08:14Z",
-            "ActivityState": "Active"
-        }
-    ],
-    "VoucherAssignments": [
-        {
-            "VoucherId": "fe568bbd-1ecb-4bb2-bf77-96c3698de20d",
-            "RateId": "6639eaa9-bbe0-46c0-94a2-aa5d00a2353c"
-        },
-        {
-            "VoucherId": "fe568bbd-1ecb-4bb2-bf77-96c3698de20d",
-            "RateId": "61133a42-41d2-4e46-b5b0-ab1701268b75"
-        }
-    ],
-    "Rates": null,
-    "Companies": null,
-    "Cursor": "fe568bbd-1ecb-4bb2-bf77-96c3698de20d"
+  "Vouchers": [
+    {
+      "Id": "fe568bbd-1ecb-4bb2-bf77-96c3698de20d",
+      "EnterpriseId": "3fa85f64-5717-4562-b3fc-2c963f66afa6",
+      "ServiceId": "bd26d8db-86da-4f96-9efc-e5a4654a4a94",
+      "Type": "Public",
+      "Name": "Weekend Voucher",
+      "CreatedUtc": "2018-11-29T08:17:05Z",
+      "UpdatedUtc": "2020-10-30T13:38:49Z",
+      "ActivityState": "Active",
+      "CompanyId": "3506994b-3c0b-49ba-9f57-ac4700641440",
+      "TravelAgencyId": null,
+      "ExternalIdentifier": "VCHR-278"
+    }
+  ],
+  "VoucherCodes": [
+    {
+      "VoucherId": "fe568bbd-1ecb-4bb2-bf77-96c3698de20d",
+      "Value": "TEST-VOUCHER-CODE",
+      "ValidityStartUtc": null,
+      "ValidityEndUtc": null,
+      "CreatedUtc": "2020-10-30T13:37:16Z",
+      "UpdatedUtc": "2020-10-30T13:37:16Z",
+      "ActivityState": "Active"
+    },
+    {
+      "VoucherId": "fe568bbd-1ecb-4bb2-bf77-96c3698de20d",
+      "Value": "05400269B23A59C649E4",
+      "ValidityStartUtc": "2020-10-09T22:00:00Z",
+      "ValidityEndUtc": "2020-10-09T22:00:00Z",
+      "CreatedUtc": "2020-10-09T15:08:14Z",
+      "UpdatedUtc": "2020-10-09T15:08:14Z",
+      "ActivityState": "Active"
+    }
+  ],
+  "VoucherAssignments": [
+    {
+      "VoucherId": "fe568bbd-1ecb-4bb2-bf77-96c3698de20d",
+      "RateId": "6639eaa9-bbe0-46c0-94a2-aa5d00a2353c"
+    },
+    {
+      "VoucherId": "fe568bbd-1ecb-4bb2-bf77-96c3698de20d",
+      "RateId": "61133a42-41d2-4e46-b5b0-ab1701268b75"
+    }
+  ],
+  "Rates": null,
+  "Companies": null,
+  "Cursor": "fe568bbd-1ecb-4bb2-bf77-96c3698de20d"
 }
 ```
 
 | Property | Type | Contract | Description |
 | :-- | :-- | :-- | :-- |
-| `Vouchers` | array of [Voucher](#voucher) | optional | Details about vouchers added to the system. |
-| `VoucherCodes` | array of [Voucher code](vouchercodes.md#voucher-code) | optional | Information about voucher codes used by customers. |
-| `VoucherAssignments` | array of [Voucher assignment](#voucher-assignment) | optional | The assignments between vouchers and [Rates](rates.md#rate). |
+| `Vouchers` | array of [Voucher](vouchers.md#voucher) | optional | Details about vouchers added to the system. |
+| `VoucherCodes` | array of [Voucher code](vouchers.md#voucher-code) | optional | Information about voucher codes used by customers. |
```

Comment:
issue: This should link to https://mews-systems.gitbook.io/connector-api/operations/vouchercodes#voucher-code

Reply:
[fixed here](https://github.com/MewsSystems/gitbook-connector-api/pull/622/commits/ff79469783614a29422ef1a72df45b800e02a4a2)